lib/webdriver.py

```diff
@@ -122,34 +127,52 @@ def get_reservations(self, account_monitor: AccountMonitor) -> List[JSON]:
         reservations = self._fetch_reservations(driver)
 
         driver.quit()
+        if self.display is not None:
+            self.display.stop()
+            logger.debug("Stopped virtual display successfully")
         return reservations
 
     def _get_driver(self) -> Driver:
+        is_docker = os.environ.get("AUTO_SOUTHWEST_CHECK_IN_DOCKER") == "1"
+
+        if is_docker:
+            try:
+                self.display = Display(visible=0, size=(1440, 1880))
+                self.display.start()
+                if self.display.is_alive():
+                    logger.debug("Started virtual display successfully")
+                else:
+                    logger.debug("Started virtual display but is not active")
+            except Exception as e:
+                logger.debug(f"Failed to start display: {e}")
+
         logger.debug("Starting webdriver for current session")
         browser_path = self.checkin_scheduler.reservation_monitor.config.browser_path
 
-        driver_version = "mlatest"
-        if os.environ.get("AUTO_SOUTHWEST_CHECK_IN_DOCKER") == "1":
-            # This environment variable is set in the Docker image. Makes sure a new driver
-            # is not downloaded as the Docker image already has the correct driver
-            driver_version = "keep"
-
-        driver = Driver(
-            binary_location=browser_path,
-            driver_version=driver_version,
-            headless=True,
-            uc_cdp_events=True,
-            undetectable=True,
-            is_mobile=True,
-        )
+        driver_options = {
+            "binary_location": browser_path,
+            "uc_cdp_events": True,
+            "undetectable": True,
+            "incognito": True,
+        }
+
+        if is_docker:
+            driver_options["driver_version"] = "keep"
+            driver_options["headed"] = True
+        else:
+            driver_options["driver_version"] = "mlatest"
+            driver_options["headless"] = True
+
+        driver = Driver(**driver_options)
         logger.debug("Using browser version: %s", driver.caps["browserVersion"])
 
         driver.add_cdp_listener("Network.requestWillBeSent", self._headers_listener)
+        driver.delete_all_cookies()
 
-        logger.debug("Loading Southwest home page (this may take a moment)")
-        driver.open(BASE_URL)
+        logger.debug("Loading Southwest check-in page (this may take a moment)")
```

Comment:
The `BASE_URL` is still the Southwest home page, right? Or was that intended to change to the check-in page.

Reply:
We first navigate to homepage and then by using click going to check-in page. So the end destination is 'Check-In'